agency/README.md

```diff
@@ -8,17 +8,26 @@ This specification contains a collection of RESTful APIs used to specify the dig
 
 ## Table of Contents
 
-* [General Information](#general-information)
-  * [Versioning](#versioning)
-  * [Responses and Error Messages](#responses-and-error-messages)
-  * [Authorization](#authorization)
-* [Vehicles](#vehicles)
-* [Vehicle - Register](#vehicle---register)
-* [Vehicle - Update](#vehicle---update)
-* [Vehicle - Events](#vehicle---event)
-* [Vehicle - Telemetry](#vehicle---telemetry)
-* [Telemetry Data](#telemetry-data)
-* [Stops](#stops)
+- [Mobility Data Specification: **Agency**](#mobility-data-specification-agency)
```

Comment:
Remove page name and TOC link, and reduce indentation for all other items.

Reply:
Resolved in 9b8b0a3